sailfishusermanagerinterface.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2020 Jolla Ltd.
+ * Copyright (c) 2020 Open Mobile Platform LLC.
+ *
+ * All rights reserved.
+ *
+ * BSD 3-Clause License, see LICENSE.
+ */
+
+#ifndef SAILFISHUSERMANAGERINTERFACE_H
+#define SAILFISHUSERMANAGERINTERFACE_H
+
+#include <QString>
+#include <QDBusArgument>
+
+#define SAILFISH_USERMANAGER_DBUS_INTERFACE "org.sailfishos.usermanager"
```

Comment:
QStringLiteral instead, please. The same on the line below.

Reply:
Just a note that if there ever are sub-interfaces, then the string literal is nicer, as they can be concatenated in the compiler.

Reply:
Actually better avoid QStringLiteral or QString in headers, those would create separate instances by all the files including the header.

Reply:
Used defines to avoid warnings from the compiler, didn't yet find a way to use QStringLiteral without complaints,

Reply:
Well, pvuorela's comment is right that there could be more instances then. I did manage to use these inside QStringLiteral without problems on my code though. Maybe not too big of an issue then.